handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslClientContext.java

```diff
@@ -90,71 +88,82 @@ static OpenSslSessionContext newSessionContext(ReferenceCountedOpenSslContext th
             throw new IllegalArgumentException(
                     "Either both keyCertChain and key needs to be null or none of them");
         }
+        OpenSslKeyMaterialProvider keyMaterialProvider = null;
         try {
```

Comment:
note that most of the changes here are just because I added another level of `try {...} catch { ...}`